client/cypress/integration/app.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { AppPage } from '../support/app.po';
+
+const page = new AppPage();
+
+describe('App', () => {
+  beforeEach(() => page.navigateTo());
+
+  it('Should have the correct title', () => {
+    page.getAppTitle().should('contain', 'CSCI 3601 Iteration Template');
+  });
+
+  it('The sidenav should open, navigate to "Users" and back to "Home"', () => {
+    // Before clicking on the button, the sidenav should be hidden
+    page.getSidenav()
+      .should('be.hidden')
+      .and('not.be.visible');
+
+    page.getSidenavButton().click()
+      .should('not.be.hidden')
+      .and('be.visible');
+
+    page.getNavLink('Users').click();
+    cy.url().should('match', /.*\/users$/);
+    page.getSidenav()
+      .should('be.hidden')
+      .and('not.be.visible');
+
+    page.getSidenavButton().click();
+    page.getNavLink('Home').click();
+    cy.url().should('match', /.*\/$/);
```

Comment:
What about a regex like this…?

```suggestion
    cy.url().should('match', /^https?:\/\/[^\/]+\/$/);
```

That way, it should only match the home page, not some other URL that ends with a slash. (Altough, I don't know, maybe the home page is the only URL that ends with a slash ¯\\\_(ツ)\_/¯)

Reply:
(If the latter, I might recommend this change instead. Up to you! 🙂)

```suggestion
    cy.url().should('match', /\/$/);
```

Reply:
We should really capture both localhost:4200 with and without a trailing slash.

Reply:
```suggestion
    cy.url().should('match', /^https?:\/\/[^\/]+\/?$/);
```
Would this achieve that @helloworld12321?

Reply:
Nice, looks good! 🙂